### PR TITLE
feat: update config Geo

### PR DESCRIPTION
**Pull request by arturmon**
Added support for downloading Geoip database from maxmind.

### CODE REVIEW

File: charts/sentry/templates/pvc-geoip.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+metadata:
+  name: data-sentry-geoip
+  labels:
+    app: sentry
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  accessModes:
+    - ReadWriteOnce
```
Comment (adonskoy) [inline charts/sentry/templates/pvc-geoip.yaml]: In this volume access mode, all pods must be on the same node. You can change it to RWM, but then you need to specify in the readme that the storage provider must support this mode

Or download the database every time and save it to an ephemeral volume via the init container

Comment (arturmon) [reply charts/sentry/templates/pvc-geoip.yaml]: @adonskoy  I made rwx with a note in the readme.  The EmptyDir storage will need to be initialized in each service of the container, and plus it still uses memory, which can, in theory, lead to an oom kill. Which is not very good.

**Comment by sdernbach-ionos** [comment]
I would love to see this merged as I think it is much easier to setup geoip like this then creating all the storage stuff on your own 👍🏻

**Comment by Mokto** [comment]
It has been implemented upstream.

**Pull request closed without merging**